.github/workflows/component-build-images.yml

```
@@ -111,6 +111,10 @@ jobs:
             tag_suffix: shippingservice
             context: ./
             setup-qemu: true
+          - file: ./src/flagd-ui/Dockerfile
+            tag_suffix: shippingservice
```

```suggestion
            tag_suffix: flagdui 
```
This should be flagdui (without '-' to follow the other images, and because helm chart removes '-' in https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-demo/templates/_objects.tpl#L52

---

Thanks @klucsik !! It's fixed now :)